src/styles/common/_global.scss

```diff
@@ -0,0 +1,65 @@
+/**
+ * TYPOGRAPHY SPACE
+ */
+$primary-font: arial;
+$secondary-font: sans-serif;
+
+h1,
```

Comment:
this is a question more so than a comment - is it better practice to put each element on its own line like this than having a comma separated string with all on one line?

Reply:
vscode auto complete did that. i just turn it off.